### ISSUE TITLE
Stemming "فاطفالهم"

### ISSUE THREAD

**Issue by LeviticusMB** [assem-ch/arabicstemmer#22]
First of all, I do not speak any Arabic whatsoever, but we're using Snowball in one of our products and we're now adding support for Arabic.

According to the [README](https://github.com/assem-ch/arabicstemmer/blob/a02b4c415ad5431b573560e18c090aa307cc8d70/README.md#results) (and Google Translate seems to agree with this), the stem of "فاطفالهم" should be "اطفال". However, both our code, after linking the current master of this project, and http://arabicstemmer.com/ stems this word as "فاطفال".

Is this a bug or am I doing something wrong?

**Comment by assem-ch** [comment]
Yes, something is wrong, it should be اطفال, we ll investigate it , thanks

**Comment by LeviticusMB** [comment]
Any news?
